draft-ietf-quic-ack-frequency.md

```diff
@@ -342,19 +342,16 @@ see {{out-of-order}}.
 
 It is important to receive timely feedback after long idle periods,
 e.g. update stale RTT measurements. When no acknowledgment has been sent in
-over one smoothed round trip time, receivers are encouraged to send an
-acknowledgment soon after receiving an ack-eliciting packet. This is not an
-issue specific to this document, but the mechanisms specified herein could
-create excessive delays.
+over one smoothed round trip time ({{Section 5.3 of QUIC-RECOVERY}}),
+receivers are encouraged to send an acknowledgment soon after receiving
+an ack-eliciting packet. This is not an issue specific to this document,
+but the mechanisms specified herein could create excessive delays.
 
 ## Response to Out-of-Order Packets {#out-of-order}
 
 As specified in {{Section 13.2.1 of QUIC-TRANSPORT}}, endpoints are expected to
-send an acknowledgment immediately on receipt of a reordered ack-eliciting
-packet with a smaller packet number than the highest-numbered ack-eliciting packet
-or with a higher packet number when there are missing packets between that packet
-and the highest-numbered ack-eliciting packet.
-This extension modifies that behavior when an ACK_FREQUENCY frame with
+send an immediate acknowledgment upon receipt of a reordered ack-eliciting
+packet. This extension modifies that response after an ACK_FREQUENCY frame with
```

Comment:
The text that you removed was added based on issue #247 as a requested clarification.

Reply:
Thanks for pointing that out.

I worry that this description is not specific enough, since it contains less text than RFC9000, and I would prefer to not restate what a reordered ack-eliciting packet is here.

Reply:
```suggestion
send an immediate acknowledgment upon receipt of a reordered ack-eliciting
packet or if a gap in the packet numbers was detected.
This extension modifies that response after an ACK_FREQUENCY frame with
```
The point in the issue was that packets gaps are not covered by this sentence. Does this phrasing maybe work?

Reply:
Re-reading the issue, I see how my text isn't sufficient, but I think the existing text in the draft isn't quite clear enough.